sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-processing-steps/e-cert-steps-utils.ts

```diff
@@ -0,0 +1,47 @@
+import { BC_FUNDING_TYPES } from "@sims/services/constants";
+import {
+  DisbursementValue,
+  OfferingIntensity,
+  RestrictionActionType,
+} from "@sims/sims-db";
+import { EligibleECertDisbursement } from "../disbursement-schedule.models";
+
+/**
+ * Check active student restrictions by its action type
+ * in an eligible disbursement.
+ * @param studentRestrictions student restrictions.
+ * @param actionType action type.
+ * @returns the first restriction of the requested
+ * action type.
+ */
+export function getRestrictionByActionType(
+  eCertDisbursement: EligibleECertDisbursement,
```

Comment:
We can but since the helper is dedicated to the steps and all steps receive the eCertDisbursement I thought that would be cleaner in this way. I would be open to change.